docs/localization-translation/README.md

```diff
@@ -0,0 +1,181 @@
+# Translating WooCommerce
+
+WooCommerce is already translated into several languages and is translation-ready right out of the box. All that’s needed is a translation file for your language.
+
+There are several methods to create a translation, most of which are outlined in the WordPress Codex. In most cases you can contribute to the project on [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/woocommerce/).
+
+To create custom translations:
+
+-   For a simple approach, choose the [Loco Translate](https://wordpress.org/plugins/loco-translate/) plugin.
+-   If you're more experienced, consider [Poedit ](https://poedit.net/).
+
+This document provides details on both methods.
+
+## Set up WordPress in your language
+
+To set your WordPress site's language:
+
+1. Go to `WP Admin » Settings » General` and adjust the `Site Language`.
+2. Go to `WP Admin » Dashboard » Updates` and click the `Update Translations` button.
+
+Once this has been done, the shop displays in your locale if the language file exists. Otherwise, you need to create the language files (process explained below).
+
+## Contributing your localization to core
+
+We encourage contributions to our translations. If you want to add translated strings or start a new translation, simply register at WordPress.org and submit your translations to [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/woocommerce/) for approval.
```

Comment:
The mismatch between the URL stub (translate.wordpress.org) and the full path it links to can be misleading - especially given the perceived semantic meaning of the visible URL "translate...WordPress". While less apparent when viewing the markdown, the rendered view had me thinking this wasn't going to link to the right place.

When URL is a label, I think it'd be better to either include the full path as the label, or link "submit your translations for approval" as the linked label. For example:

```md
...and [submit your translations for approval](https://translate.wordpress.org/projects/wp-plugins/woocommerce/).
```

_This comment applies to elsewhere in the doc where the similar pattern is found._

Reply:
Thanks for pointing this out. As we're showing the URLs in different contexts, I decided to show the full URL in all cases.